recipes/omnipath/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "Omnipath" %}
+{% set version = "1.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0]|lower }}/{{ name|lower }}/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: 16e0dd4ede915d9650d03905f71c44168299f5556e929c17631da5bfff3a6023
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+  skip: true  # [py<37]
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - python
+    - attrs >=20.2.0
+    - docrep >=0.3.1
+    - inflect >=4.1.0
+    - packaging
+    - pandas >=1.2.0
+    - requests >=2.24.0
+    - tqdm >=4.51.0
+    - typing_extensions >=3.7.4.3
+    - urllib3 >=1.26.0
+    - wrapt >=1.12.0
+
+test:
+  imports:
+    - omnipath
```

```suggestion
test:
  imports:
    - Omnipath
  commands:
    - pip check
  requires:
    - pip
```
This makes sure that the deps are correct.